lib/wovnrb/store.rb

```diff
@@ -30,7 +30,7 @@ def reset
           'url_pattern' => 'path',
           'url_pattern_reg' => "/(?<lang>[^/.?]+)",
           'query' => [],
-          'api_url' => 'https://api.wovn.io/v0/values',
```

Comment:
can you use "api.wovn.io" as default setting?
by seeing this function, we can easily understand default.